pyrisco/local/risco_socket.py

```diff
@@ -28,8 +29,9 @@ async def connect(self):
     self._cmd_id = 0
     try:
       self._semaphore = asyncio.Semaphore(self._max_concurrency)
-      self._futures = [None for i in range(MIN_CMD_ID,MIN_CMD_ID + MAX_CMD_ID)]
+      self._futures = [None for i in range(MIN_CMD_ID, MIN_CMD_ID + MAX_CMD_ID)]
       self._reader, self._writer = await asyncio.open_connection(self._host, self._port)
+      await asyncio.sleep(self._communication_delay)
```

Comment:
```suggestion
      if self. _communication_delay > 0:
        await asyncio.sleep(self._communication_delay)
```
That way we don't needlessly yield if there's no need to.

Reply:
Yeah, I was also considering doing it that way but since there is little to no overhead and it is just on the opening of the connection I kept it like this. 
Merging you suggestion!